src/main.js

```diff
@@ -4,6 +4,8 @@ var coverTitle = document.querySelector('.cover-title')
 var coverTagLine = document.querySelector('.tagline')
 var priceTag = document.querySelector('.price-tag')
 var randomBtn = document.querySelector('.random-cover-button')
+var makeNewBtn = document.querySelector('.make-new-button')
+var makeOwnPage = document.querySelector('form')
```

Comment:
I'd prefer it if we could merge just the iteration 0 stuff and save the iteration 1 WIP stuff for a later commit / merge. It's not a big deal worth time traveling over on this project imo, but let's try to keep it in mind for future projects!